libs/unity/library/Editor/PeerConnectionEditor.cs

```diff
@@ -209,6 +212,23 @@ public override void OnInspectorGUI()
         {
             serializedObject.Update();
 
+#if UNITY_WSA
+            if (!PlayerSettings.WSA.GetCapability(PlayerSettings.WSACapability.Microphone))
+            {
+                EditorGUILayout.HelpBox("The UWP player is missing the Microphone capability. Currently on UWP the native WebRTC implementation always tries to"
+                    + " open the microphone while initialzing the audio subsystem at startup. Not granting access will fail initialization, and generally crash the app."
+                    + " Add the Microphone capability in Project Settings > Player > UWP > Publishing Settings > Capabilities.", MessageType.Error);
+                if (GUILayout.Button("Open Player Settings"))
+                {
+                    SettingsService.OpenProjectSettings("Project/Player");
+                }
+                if (GUILayout.Button("Add Microphone Capability"))
+                {
+                    PlayerSettings.WSA.SetCapability(PlayerSettings.WSACapability.Microphone, true);
+                }
+            }
+#endif
+
```

Comment:
nit: why both here and in MicrophoneSource?

Reply:
As explained in the help message, because of some "bug"/"feature" in the Audio Module implementation which currently checks audio devices on `libwebrtc` library startup, which crashes on UWP if audio permission is not granted. In comparison the Video Module correctly only accesses the webcam when creating a video track source, so no need to force that permission.

Reply:
Right, didn't read the message carefully enough.